### PR TITLE
set from_logits True in loss function

### PR DESCRIPTION
**Pull request by AshokKumar2525**
By settinng the from_logits True in loss function it will optimize the model even more.

### CODE REVIEW

**Comment by AshokKumar2525** [comment]
hello @MahraibFatima, @PRIYANSHU2026 ! check it once. I have another issue assigned which is completed. if you check and merge it i will pull request that too.

**Comment by MahraibFatima** [comment]
add `readme.md` and `requirements.txt` for your project with proper details.

**Comment by AshokKumar2525** [comment]
@MahraibFatima the project already has readme file and requirements.txt there are no changes in them, the only change is in model.py
